src/browser_actions/main.css

```diff
@@ -4,6 +4,22 @@ html {
     background-color: rgb(249, 249, 251);
 }
 
+html.dark-theme, html.dark-theme * {
+    background-color: #1c1c1c !important;
```

Comment:
So that the `<li>` elements don't keep their default `rgb(255, 255, 255)` color.
![image](https://user-images.githubusercontent.com/23298862/87933491-03731400-ca96-11ea-8170-12e14c73afe5.png)